challenge_2/python/bryantpq/search_item.py

```diff
@@ -0,0 +1,24 @@
+def search_item(a):
+    d = {}
+    for i in a:
+        if i in d:
+            d[i] += 1
```

Comment:
There's a [Counter](https://docs.python.org/2/library/collections.html#collections.Counter) class just for this.